### PR TITLE
Add support for Solarflare SFN7122F 10Gb NIC

### PR DESCRIPTION
**Pull request by shir474**
Signed-off-by: shir474 <shir474@users.noreply.github.com>

### CODE REVIEW

**Comment by ipxe-devel** [comment]
On Mon, Mar 30, 2020 at 02:21:16AM -0700, Martin Habets wrote:
> @habetsm-xilinx approved this pull request.
> 
> Looks good to me. Thanks!

But the  E-mail address looks strange  ...

> _______________________________________________
> ipxe-devel mailing list
> ipxe-devel@lists.ipxe.org
> https://lists.ipxe.org/mailman/listinfo/ipxe-devel



From: shir474 <shir474@users.noreply.github.com>
Date: Thu, 26 Mar 2020 18:46:21 -0400
Subject: [PATCH] Add support for Solarflare SFN7122F 10Gb NIC

Signed-off-by: shir474 <shir474@users.noreply.github.com>
---
 src/drivers/net/sfc/sfc_hunt.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/drivers/net/sfc/sfc_hunt.c
b/src/drivers/net/sfc/sfc_hunt.c
index dd5f7043f..df5e7a85f 100644
--- a/src/drivers/net/sfc/sfc_hunt.c
+++ b/src/drivers/net/sfc/sfc_hunt.c
@@ -1314,6 +1314,7 @@ const struct efx_nic_type hunt_nic_type = {
 };
 
 static struct pci_device_id hunt_nics[] = {
+	PCI_ROM(0x1924, 0x0903, "SFC9120", "Solarflare SFN7xxx Adapter", 0),
 	PCI_ROM(0x1924, 0x0a03, "SFC9220", "Solarflare SFN8xxx Adapter", 0),
 	PCI_ROM(0x1924, 0x0b03, "SFC9250", "Solarflare X25xx Adapter", 0),
 };


Regards
Geert Stappers
-- 
Silence is hard to parse

**Comment by mcb30** [comment]
I think this PCI ID is included in commit 78ca890c3
